Sources/ObjCSupport/LOKButtonLayout.swift

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import UIKit
+
+@objc public class LOKButtonLayout: LOKBaseLayout {
+    @objc public init(type: LOKButtonLayoutType,
+                      title: String?,
+                      image: UIImage?,
+                      imageSize: CGSize,
+                      font: UIFont? = nil,
+                      contentEdgeInsets: NSValue?, // UIEdgeInsets
```

Comment:
Instead of `NSValue`, we might consider wrapping it a class. To compare with `NSValue`, it would be easier to understand and configure.